.github/workflows/project-updater.yml

```
@@ -0,0 +1,23 @@
+name: Update projects when an issue/PR is created/labeled
+
+on:
+  issues:
+    types:
+      - opened
+      - labeled
+  pull_request:
+    types:
+      - opened
+      - labeled
+
+jobs:
+  add-to-project:
+    name: Add to the Release and Deferred blocker project
```

```suggestion
    name: Add to the Release and Deferred Blocker project
```

Consistent case makes it a little more clear the intended meaning is "Add to the (Release and Deferred Blocker) project", rather than "Add to the (Release) and (Deferred blocker) project[s]" (as I originally interpreted it)